src/scripts/pagerduty.coffee

```diff
@@ -808,6 +808,8 @@ module.exports = (robot) ->
             ]
           return cb(users, schedule)
         cb(json.entries[0].user, schedule)
+      else
+        cb(null, schedule)
```

Comment:
@nadade would you mind adding this `else` block also to other variants of `if json.entries.length > 0 ...` too please? Thank you.